### ISSUE TITLE
Some questions about model

### ISSUE THREAD

**Issue by huangcimu** [JiaRenChang/PSMNet#164]
hello!
     In other issues,you said your publised model is trained for 1000 epochs  using all 2015 KITTI traning dataset.In the process of training, do you choose all 200 sets of data in the KITTI2015 training set for training, and then use the 40 sets of data in the training set as the validation set. The selected model is the model with the smallest verification error?

**Comment by JiaRenChang** [comment]
@huangcimu 
The published model is adopted from the final epoch.

**Comment by huangcimu** [comment]
Thank you for your answer. So do you mean that you directly train the 200 data sets without the validation set, and then select the model produced by the last epoch is your model

**Comment by huangcimu** [comment]
Thank you for your answer. So do you mean that you directly train the 200 data sets without the validation set, and then select the model produced by the last epoch is your published model

**Comment by JiaRenChang** [comment]
@huangcimu 
Yes.
